stack.py

```diff
@@ -61,6 +61,17 @@ def __eq__(self, other):
 }
 
 
+def split_code(text):
+    lines = (line.strip() for line in text.split('\n'))
+    # We want to skip all comment lines
+    lines = (line for line in lines if line and line[0] != ':')
+    # Here lines is an iterable of lines that are not comments
+    items = itertools.chain(*(line.split(';') for line in lines))
+    items = (item.strip() for item in items)
+    # We want to skip all empty strings
+    return (item for item in items if item)
```

Comment:
I don't know if this is intended behavior but `list(s.split_code("add : nothing"))` returns `['add : nothing']`